### PR TITLE
Xivo 2018

### PR DESCRIPTION
**Pull request by mwolff44**

### CODE REVIEW

File: contrib/export_xivo_2018.sh [hunk 1]
```diff
@@ -0,0 +1,407 @@
+#!/bin/bash
+# Copyright 2021 The Wazo Authors  (see the AUTHORS file)
+# SPDX-License-Identifier: GPL-3.0-or-later
+
+set -e
+set -u  # fail if variable is undefined
+set -o pipefail  # fail if command before pipe fails
+
+# This script should be executed as user postgres
```
Comment (pc-m) [inline contrib/export_xivo_2018.sh:9]: Could you add a comment describing the difference between the export_xivo_2018 and export_xivo scripts? To be able to spot the differences between the 2 versions easily when/if we are going to modify the scripts

**Review by pc-m — CHANGES_REQUESTED** [review]

File: contrib/export_xivo_2018.sh (hunk 1 above)
Comment (mwolff44) [reply contrib/export_xivo_2018.sh:9]: mainly some useful debug information and cleaning SQL queries

**Review by pc-m — APPROVED** [review]

File: contrib/export_xivo_2018.sh (hunk 1 above)
Comment (pc-m) [reply contrib/export_xivo_2018.sh:9]: Should we remove the old version then?

Comment (mwolff44) [reply contrib/export_xivo_2018.sh:9]: i will do another PR, but i prefer to keep it for the moment

**Pull request merged**